src/goto-programs/remove_asm.cpp

```diff
@@ -129,167 +184,274 @@ void remove_asmt::process_instruction(
   const irep_idt &flavor=code.get_flavor();
 
   if(flavor==ID_gcc)
+    process_instruction_gcc(code, dest);
+  else if(flavor == ID_msc)
+    process_instruction_msc(code, dest);
+  else
+  {
+    // ignore
```

Comment:
I'd rather see `UNIMPLEMENTED` or the likes here. Those silent failures have bitten us before.

Reply:
Moved to PR #2714